applications/TheLostDiamond_BjornOmsland.md

```diff
@@ -0,0 +1,208 @@
+
+150 million people are hunting virtual monsters on PokémonGo. By utilizing EOS, we have the opportunity and advantage to replace virtual monsters with real money in a new real-world adventure game that is fun and exciting for everyone. This is a great way to teach non-tech people about the EOS-ecosystem and the possibilities in web3 applications.
+
+For people to adapt a new technology in mass, it needs to improve their lives by an order of magnitude. 
+It’s not an easy task to motivate non-tech people to create an EOS-account, download and learn how to use Anchor sign-in or create a Coinbase account. It’s a steep learning curve.
+
+TheLostDiamond solves this by having a $1,000 dollar lost diamond as incentive for participating. Also, the concept of a treasure hunt is fun and easy to understand for everyone. 
+
+We will take a $7 dollar monthly subscription fee to participate and the smartcontract returns 80-90% back to the Play2Earn model to fund the lost diamonds. A true Web3 application where everyone can create, own and monetize their content.
+
+We are aiming for 1 million users (0.65% market share from PokémonGo) within four years.
+
+TheLostDiamond will be self-sustaining at 5,000 active users per month. 
+
+Until we reach a self-sustaining level, we need to fund the $1,000 dollar lost diamonds from sponsors, grants, investors and/or entry fees from other single-event races. 
+
+The current value of The Lost Diamond is $440 dollars. The Lost Diamond race is online, if you sign in with Anchor - and if you find The Lost Diamond, you and the checkpoint creator will both get $400 dollars - instantly paid out to your EOS-account.
+
+Our first milestone is to prove that we can onboard participants, that pay a $7 dollar monthly subscription fee, to the MVP and obtain a 22% monthly growth rate over several months. Showing detailed traction is crucial for future funding from e.g. investors, sponsors, ENF and/or Pomelo. 
+
+The first $1,000 dollar diamond will be announced in the beginning of May 2022 and will be self-funded.
+
+What is the definition of a participant: A participant that pay a $7 dollar monthly subscription fee to participate in TheLostDiamond adventure race.
+
+What is the current baseline: 0 participants. We have not started marketing.   
+
+What is the explicit target number of users given the baseline:
+First milestone is to get 100 non-tech users to participate.
+
+Then our goal is a 22% monthly growth:
+- Month  1: 100 users + 22% growth = 122 active users per month
+- Month  2: 122 users + 22% growth = 148 active users per month
+- Month  3: 148 users + 22% growth = 180 active users per month
+- Month  6: 325 users + 22% growth = 396 active users per month
+- Month  12: 1,068 users + 22% growth = 1,303 active users per month
+- Month  24: 11,597 users + 22% growth = 14,148 active users per month
+- Month  36: 126,065 users + 22% growth = 153,799 active users per month
```

Comment:
This seems like an unusually high expectations to 10x the active users per months.

Going from 1K DAU => 10K DAU => 100K DAU

Anything above 10K DAU will require a serious budget to maintain and operate this type of application.

Reply:
It's not 10x active users per months. It's 22% monthly growth. 

I simplified the table by only showing Month 6 (half year), Month 12 (one year), Month 24 (two years), Month 36 (three years). I have now included the full table with all 36 months in the latest commit.

Yes, anything above 10K DAU will require a serious budget. And also several thousands checkpoints all over the world. More details about the total budget and financial model can be found in the Pitch Deck Appendix (link in the Additional Information section)